pages/common/passwd.md

```diff
@@ -10,10 +10,6 @@
 
 `passwd {{new_password}}`
```

Comment:
I think this example is invalid too
```suggestion
 `passwd {{username}}`
```

Reply:
The description for it too would be incorrect as well, instead of reading `- Change the password of the current user:` it should be `- Change the password of any user:`, is it possible you could suggest this change or commit it to my repo? I'm unsure if I could do it lol

Reply:
I'm gonna see if I can apply this and stuff, new to github so I'm learning all this lol.